src/_cffi_src/openssl/objects.py

```diff
@@ -9,6 +9,14 @@
 """
 
 TYPES = """
+typedef struct {
+    int type;
+    int alias;
+    const char *name;
+    const char *data;
+} OBJ_NAME;
```

Comment:
the fields alias and name are used. CPython stdlib uses them to find out which hashing algorithms are available on the target platform. (it calls `OBJ_NAME_do_all(OBJ_NAME_TYPE_MD_METH, ...)`)